packages/edit-post/src/index.js

```diff
@@ -96,7 +96,7 @@ export function initializeEditor(
 	);
 	registerCoreBlocks();
 	if ( process.env.GUTENBERG_PHASE === 2 ) {
-		__experimentalRegisterExperimentalCoreBlocks( settings );
+		__experimentalRegisterExperimentalCoreBlocks( settings, { postType } );
```

Comment:
what if we passed a list of blocks to filter out instead of passing the block type?

`__experimentalRegisterExperimentalCoreBlocks( settings, blockExcludeList )`?

Reply:
We're passing the post type (rather than the block type). Personally, I'd rather not pass a deny list of blocks, since I'd like to keep the interface at least somewhat generic, and having a block context as second argument seems more universal than a block deny list.

Reply:
thinking about this as if it is "block context" makes sense to me. I didn't originally catch on to that.